### PR TITLE
Add new vpc/subnets for scicomp.

### PR DESCRIPTION
**Pull request by xschildw**

### CODE REVIEW

File: templates/base_vpc.yaml [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    Description: The VPC subnet prefix (i.e. 10.40)
+    Type: String
+  SubnetZone:
+    Description: Availability zone for public subnet
+    Type: String
+    AllowedValues:
+      - us-east-1a
+      - us-east-1b
+      - us-east-1c
+      - us-east-1d
```
Comment (zaro0508) [inline templates/base_vpc.yaml]: what about us-east-1e and us-ease-1f?  
no support for non us-east regions?

File: templates/user_vpc.yaml [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    Description: The VPC subnet suffix (e.g. 0.2/8)
+    Type: String
+  SubnetZone:
+    Description: Availability zone for subnet
+    Type: String
+    AllowedValues:
+      - us-east-1a
+      - us-east-1b
+      - us-east-1c
+      - us-east-1d
```
Comment (zaro0508) [inline templates/user_vpc.yaml]: 1e and 1f?

File: templates/user_vpc.yaml [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    Default: us-east-1a
+  SubnetType:
+    Description: Type of the subnet (public or private)
+    Type: String
+    AllowedValues:
+      - public
+      - private
+Conditions:
+  PublicSubnetType: !Equals [public, !Ref SubnetType]
+  PrivateSubnetType: !Equals [private, !Ref SubnetType]
```
Comment (zaro0508) [inline templates/user_vpc.yaml]: [optional] . maybe you only need to use one condition here? 

CreatePublicResources: !Equals [public, !Ref SubnetType]
then use "!Not CreatePublicResources" for private

Comment (xschildw) [reply templates/user_vpc.yaml]: I think I tried that and got an error. Maybe I missed something...

**Comment by zaro0508** [comment]
Do we still need this?

**Pull request closed without merging**